mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -51,48 +81,71 @@ CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 	METHOD method_8219 getProtectionAmount (Ljava/lang/Iterable;Lnet/minecraft/class_1282;)I
 		ARG 0 equipment
 		ARG 1 source
-	METHOD method_8220 accept (Lnet/minecraft/class_1890$class_1891;Lnet/minecraft/class_1799;)V
-		ARG 0 enchantmentHandler
+	METHOD method_8220 process (Lnet/minecraft/class_1890$class_1891;Lnet/minecraft/class_1799;)V
+		ARG 0 action
 		ARG 1 stack
 	METHOD method_8221 hasVanishingCurse (Lnet/minecraft/class_1799;)Z
 		ARG 0 stack
-	METHOD method_8222 getEnchantments (Lnet/minecraft/class_1799;)Ljava/util/Map;
+	METHOD method_8222 get (Lnet/minecraft/class_1799;)Ljava/util/Map;
```

Comment:
Not sure about this change. `EnchantmentHelper.get(stack)` does not make it immediately clear that we are getting the enchantments on the stack.

Reply:
This is on par with `EnchantmentHelper.set` which already exists.